packages/eslint-plugin/typings/eslint-rules.d.ts

```diff
@@ -543,3 +543,21 @@ declare module 'eslint/lib/rules/no-extra-semi' {
   >;
   export = rule;
 }
+
+declare module 'eslint/lib/rulesinit-declarations' {
+  import { TSESLint, TSESTree } from '@typescript-eslint/experimental-utils';
+
+  const rule: TSESLint.RuleModule<
+    never,
```

Comment:
you can use the message IDs from the base rule here
https://github.com/eslint/eslint/blob/0c20bc068e608869981a10711bba88ffde1539d8/lib/rules/init-declarations.js#L89-L92

Reply:
@bradzacher I didnt get this. no other rules are having `messageIds` as type in the module.

Reply:
The ones with rules that support messageIds do have them declared, i.e.

https://github.com/typescript-eslint/typescript-eslint/blob/1e29e69b289d61107a7de67592beae331ba50222/packages/eslint-plugin/typings/eslint-rules.d.ts#L7-L14

Reply:
done